firstvoices/backend/resources/base.py

```diff
@@ -76,19 +129,3 @@ class RelatedMediaResourceMixin(resources.ModelResource):
 
     class Meta:
         abstract = True
-
-
-class AudienceMixin(resources.ModelResource):
-    exclude_from_games = fields.Field(
-        column_name="include_in_games",
-        attribute="exclude_from_games",
-        widget=InvertedBooleanFieldWidget(column="include_in_games", default=False),
```

Comment:
Removing the mixin, and adding it individually since all classes have different column name.